.github/workflows/main.yml

```diff
@@ -19,7 +16,7 @@ jobs:
           - macos-latest
           # - windows-latest
         ocaml-compiler:
-          - "4.14"
+          - 4
```

Comment:
Specify 4 instead of 4.14 to let setup-ocaml always lookup the latest OCaml 4 variant!